eslint-config-ts/package-lock.json

```diff
@@ -1,1157 +0,0 @@
-{
```

Comment:
Je tohle smazané záměrně? Není mi jasné, proč by tenhle soubor neměl existovat, stejně jako `scripts-components/in-view/package-lock.json`.

Reply:
To byl fuckup. Fixed.